dist/localization.json

```diff
@@ -95,7 +95,9 @@
     "command_failed_jail": "This Command cannot be used when in jail!",
     "command_failed_crimes": "This Command cannot be used when wanted!",
     "bypass_enabled": "Bypass has been enabled.",
-    "bypass_disabled": "Bypass has been disabled."
+    "bypass_disabled": "Bypass has been disabled.",
+    "You_only_own": "You only own {0} apartment",
+    "No_Appartments": "You don't own any appartments"
```

Comment:
Use snake_case for this, also add a "." behind your sentences.

Reply:
Done